### ISSUE TITLE
False unused namespace warning

### ISSUE THREAD

**Issue by jngbng** [clj-kondo/clj-kondo#1289]
**version**

clj-kondo v2021.05.31

**macro usage**

Maybe have something to do with [clojure.spec.alpha/keys](https://github.com/clojure/spec.alpha/blob/master/src/main/clojure/clojure/spec/alpha.clj#L416) macro

**platform**

native version of clj-kondo

**editor**

Emacs

**problem**

Namespace is reported unused when it is only used in spec.alpha/keys's argument.

```clj
(ns boxhero.hello
  (:require [cljs.spec.alpha :as s]
            [boxhero.spec :as spec]))

(s/keys :req [::spec/hello])
```

```
src/boxhero/hello.clj:3:14: warning: namespace boxhero.spec is required but never used
```

but it is used in `s/keys`.

**repro**

Please refer to problem description.

**config**

no

**expected behavior**

no warning is exepcted.

**Comment by borkdude** [comment]
Probably related to #1272

**Comment by borkdude** [comment]
Fixed on master.
